x-pack/plugin/security/cli/src/main/java/org/elasticsearch/xpack/security/cli/EnrollNodeToCluster.java

```diff
@@ -169,7 +177,7 @@ protected void execute(Terminal terminal, OptionSet options, Environment env) th
         }
 
         final UserPrincipal newFileOwner = Files.getOwner(instantAutoConfigDir, LinkOption.NOFOLLOW_LINKS);
-        if (false == newFileOwner.equals(Files.getOwner(env.configFile(), LinkOption.NOFOLLOW_LINKS))) {
+        if (shouldForce == false && (false == newFileOwner.equals(Files.getOwner(env.configFile(), LinkOption.NOFOLLOW_LINKS)))) {
```

Comment:
nit: please use `== false` or `false ==`, but not both in the same line

Reply:
multiple edits over multiple times.. that would have caught my eye too in a review